tools/torch_quant/torch_quant/quantizer.py

```diff
@@ -155,68 +138,71 @@ def get_observer_types(
 
 
 class Quantizer:
-    def __init__(self, module_filter: Optional[ModuleFilter] = None,
-                 backend: Backend = Backend.REFERENCE,
-                 device: Device = Device.X86,
-                 tracer: Optional[Tracer] = None,
-                 ) -> None:
+    def __init__(
+        self,
+        module_filter: Optional[ModuleFilter] = None,
+        backend: Backend = Backend.REFERENCE,
+        device: Device = Device.X86,
+        tracer: Optional[Tracer] = None,
+        act_ob_ctr: Optional[Callable[..., Observer]] = None,
```

Comment:
In the current design, the kinds of observer types can be changed in each call of `qat`, `ptq`, `convert`. If put the kinds of observer here, how do we switch observer types.

Reply:
Generally, the observer types in `quantize` stage should be the same as previous stage (calib, qat...). As for qat, observer types can be also changed by default or specifying.

Reply:
I am afraid not. For example, recently we are doing some quantization experiments. In these experiments, the PTQ algorithm based on mathematical statistics (e.g. gptq)will be used to initialize the quantitative parameters of the model. Then, some tuning based PTQ algorithms like brecq will be used to further improve the performance. If doing this experiments on torch-quant, the workflow should be:
```
model = some_llm_models()

gptq_quant_model = quantizer.calib(model)

do calibration

brecq_quant_model = quantizer.calib(model, w_ob_ctr=brecq_ctr)

do calibration
```

So the interface to set the ob_ctr should be available on both ptq/qat function.